dts/arm/st/u5/stm32u575Xg.dtsi

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2025 Harris Tomy
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <mem.h>
+#include <st/u5/stm32u575.dtsi>
+
+/ {
+	sram0: memory@20000000 {
+		/* SRAM1 + SRAM2 + SRAM3 */
+		reg = <0x20000000 DT_SIZE_K(768)>;
+	};
+
+	sram1: memory@28000000 {
+		/* SRAM4, low-power background autonomous mode */
+		reg = <0x28000000 DT_SIZE_K(16)>;
+	};
```

Comment:
Here also, these 2 nodes are common to stm32u575 and u585 SoCs.
Could be factorized in **stm32u575.dtsi** instead of being placed here and later replicated other stm32u575X* DTSI files.